api.planx.uk/s3/getFile.ts

```diff
@@ -0,0 +1,27 @@
+import S3 from "aws-sdk/clients/s3";
```

Comment:
Worth noting that I had to change this otherwise `tsc` would try to compile the whole sdk and the GH Action worker would run out of memory.

```
- import { S3 } from "aws-sdk";
+ import S3 from "aws-sdk/clients/s3";
```

In a separate PR I'll suggest increasing the maximum amount of memory available for the worker (just in case).

Reply:
Worth noting about the "worth noting" above that the Docker build would fail silently (ie the error message was there but it would return a Linux zero code) which made it hard for me to find the root cause at first.